addons/mod_tool/interface/panel/tools_panel.gd

```diff
@@ -3,6 +3,8 @@ class_name ModToolsPanel
 extends Control
 
 
+enum FileDialogMode {EXPORT, LINK_MOD}
```

Comment:
not sure if we want to have one thing that behaves differently for many states. Just having two separate popups with their own logic might seem a bit like repeating code, but it's probably cleaner to maintain

Reply:
done [686e03c](https://github.com/GodotModding/godot-mod-tool/pull/106/commits/686e03c4164b91ed10befcfd9f0a461c260d2567) 👍